src/common/index.ts

```diff
@@ -69,6 +69,14 @@ export function hasOwnProperty(target: unknown, property: PropertyKey) {
   return Object.hasOwnProperty.call(target, property);
 }
 
+export function isPropertySet(target: unknown, property: PropertyKey) {
+  return (
+    hasOwnProperty(target, property) &&
+    !isUndefined((target as Record<PropertyKey, unknown>)[property]) &&
+    !isNull((target as Record<PropertyKey, unknown>)[property])
```

Comment:
Is checking `isNull` correct? If the value is `null`, it means the value is set to `null`, doesn't it?

Reply:
I agree with this one - I liked the discussion version we had on linked issue where it was looking for undefined and letting that go, but null is a specific (anti-)value.

But more than anything, thanks for noticing this and taking the time to post a PR, really appreciated.